components/browser/menu/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mozilla.components.browser.menu" />
+    package="mozilla.components.browser.menu">
+
+    <application android:supportsRtl="true" />
```

Comment:
Is there a strong reason to add this here? Asking because this means we will force this on the app via manifest merge.

Reply:
I have added this to be able to test `'menu category should handle initialization with text alignment'`. Otherwise `textView.textAlignment` would always return 1 (`TEXT_ALIGNMENT_GRAVITY`)